core/dom/range.js

```diff
@@ -2818,6 +2818,107 @@ CKEDITOR.dom.range = function( root ) {
 			reference.remove();
 		},
 
+		/**
+		 * Returns an array of {@link CKEDITOR.dom.rect} elements that are represented as rectangles which are covered by ranges.
+		 * For each DOM element within the selection range there is only one rectangle.
+		 * Rectangles represent the area of the screen occupied by the elements contained within the range.
+		 *
+		 * In the following example:
+		 *
+		 *        <p><span>first { span</span><span> second text</span></p>
+		 *        <p><span>very long } span</span></p>
+		 *
+		 * Brackets represent the beginning and the end of the selection.
+		 *
+		 * Returned rectangles would be represented by areas like below:
+		 *         text [ text ] [ text text ]
```

Comment:
That doesn't render correctly in our docs. Be sure to always build docs and preview it before you publish something.